models/CreateNewPatientRequest.js

```diff
@@ -0,0 +1,83 @@
+/*
+
+Patient {
+
+  family : string
+  given : string
+  suffix : string
+  phone : {
+    value : string
+    use : enum (home, work, temp, old, mobile)
+  }
+  email : array of strings
```

Comment:
We should also be able to accommodate 2 instances of email:  primary and secondary

Reply:
This is handled - the two emails are sent in a list.